code/modules/projectiles/updated_projectiles/guns/smgs.dm

```diff
@@ -219,9 +219,14 @@
 	fire_sound = 'sound/weapons/smg_heavy.ogg'
 	current_mag = /obj/item/ammo_magazine/smg/ppsh
 	attachable_allowed = list(
-						/obj/item/attachable/flashlight)
-
+						/obj/item/attachable/flashlight,
+						/obj/item/attachable/gyro,
+						/obj/item/attachable/lasersight,
+						/obj/item/attachable/bayonet, //obligatory
+						/obj/item/attachable/suppressor
+						)
```

Comment:
Like it was pointed out, most of these attachments wouldn't go on such a SMG.
The flashlight and laser sight are ok though.

Reply:
The doublebarrel can fit a gyro and a RDS, so I don't see why they can't fit on the PPSh

Reply:
Yea, fine on those two then.